org.osgi.test.junit4/test.bndrun

```diff
@@ -30,10 +30,4 @@
 	ch.qos.logback.core,\
 	org.apache.felix.logback,\
 	slf4j.api
--runrequires: bnd.identity;id='${project.artifactId}-test'
--runbundles: \
```

Comment:
I am not sure I like removing the `-runbundles` from the bndrun file. Is that necessary or a good idea?

Reply:
I mean it's recomputed each time already. So all this does is force us to have to commit changes to source anytime we make outside changes that affect the resolution result. All I personally care about for this is that tests actually run and pass :) we'd see a failure if the resolution was ever miscalculated due to human error.

... but, I'm not strongly arguing about it... just merely annoying for this scenario :)

Reply:
re-added it...